### DISCUSSION TITLE
Upload multiple files in langflow

### DISCUSSION THREAD

**Discussion by Preet1234-ux** [langflow-ai/langflow#1846] [General]
I want to upload a full directory which contains multiples files . Which loaders component shall I use for that ? I was trying to use Directory Loader but not able to use it .

**Comment by DDT-scrip** [comment]
"Directory Loader" by rodrigosnader in langflow store is pretty easy to use

**Comment by Cristhianzl** [comment]
Hi! 👋

We are using other channels as our official means of communication with users. We apologize for the delayed response.
For faster assistance, please direct your questions to our Discord channel or open a GitHub issue, as we are much more active on these platforms.

[Langflow Discord Server](https://discord.com/invite/EqksyE2EX9)

Thank you for your understanding.

Best regards,
The Langflow Team
